docs/release-notes/Percona-Everest-1.5.0-(2025-02-07).md

```diff
@@ -0,0 +1,154 @@
+
+
+- [EVEREST-1718](https://perconadev.atlassian.net/browse/EVEREST-1718) \[UI\] Add loading bar when operators are being upgraded
+
+
+- [EVEREST-1788](https://perconadev.atlassian.net/browse/EVEREST-1788),[EVEREST-1790](https://perconadev.atlassian.net/browse/EVEREST-1790): The `everestctl namespaces remove` and `everestctl namespaces update` commands now show a help message that guides you on how to use them.
+
+
+- [EVEREST-1794](https://perconadev.atlassian.net/browse/EVEREST-1794): We have improved the description of the help text for the `--keep-namespace` flag in the `everestctl namespaces remove` command. Previously, the flag did not clearly explain that it retains the namespace in Kubernetes while only removing `everest-managed` resources, which led to confusion.
+
+- [EVEREST-1795](https://perconadev.atlassian.net/browse/EVEREST-1795): When attempting to update a namespace using `everestctl` that was created with `kubectl` (not managed by Percona Everest), the error message was unclear. It did not provide actionable steps for the user to resolve the issue. We have improved the error message to give more insights into the issue.
+
+
+## Bugs
+
+- [EVEREST-1261](https://perconadev.atlassian.net/browse/EVEREST-1261): We fixed an issue where a user who had already added a backup storage location received an incorrect error message when trying to add another one with the same bucket name and URL.
+
+
+- [EVEREST-1401](https://perconadev.atlassian.net/browse/EVEREST-1401):
+Now, when you create/edit the database cluster with sharding enabled for PSMDB, it will display the correct resources required for the specified number of shards.
+
+- [EVEREST-1537](https://perconadev.atlassian.net/browse/EVEREST-1537):
+We have resolved an issue that caused Percona Everest uninstallation to fail when attempting to delete database clusters due to a timeout.
+
+
+- [EVEREST-1581](https://perconadev.atlassian.net/browse/EVEREST-1581): The database remained in a **Deleting** state, despite all components being deleted. The issue has now been resolved.
+
+- [EVEREST-1588](https://perconadev.atlassian.net/browse/EVEREST-1588): We have fixed an issue where the PostgreSQL database was stuck in an **initializing** state after a restore.
+
+- [EVEREST-1589](https://perconadev.atlassian.net/browse/EVEREST-1589): We have fixed an issue where the MySQL database reamined stuck in an **initializing** state for 1-node cluster.
+
+- [EVEREST-1590](https://perconadev.atlassian.net/browse/EVEREST-1590) Installation is stuck
+
+- [EVEREST-1647](https://perconadev.atlassian.net/browse/EVEREST-1647): When creating a monthly schedule on day 1 at 12:00 AM (the default option when choosing **Monthly**), it led to an error for PSMDB. The issue has been resolved now.
```

Comment:
Well, this is a bit more nuanced than this. Maybe we should say that there's an issue with monthly schedules on day 1 with certain timezones and we added validations to forbid those cases.